### ISSUE TITLE
idea: make authors and keywords on dataset landing page clickable to feed the search

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandiarchive#753]
Looking at 

![image](https://user-images.githubusercontent.com/39889/122575216-82fac180-d01e-11eb-8703-49bb8f8737ce.png)

clicking on any author or keyword (ATM does nothing) could just take that entity and feed the search bar, thus allowing to quickly navigate archive for a specific author or keyword, e.g. clicking on `alpha rhythm` should lead to 

![image](https://user-images.githubusercontent.com/39889/122575398-b50c2380-d01e-11eb-872c-418ec9724c44.png)

AFAIK it should be quite easy todo for any JS savvy dev, and before some fancy faceted etc search is developed, IMHO it would greatly improve UX to navigate through the archive based on authors/keywords of interest.

notes:
- it is sufficient to "enter" into searchbar in the original dataset landing page -- it would jump into shown above search results
- unfortunately we seems to not yet support limiters in the search query, so I could not search for `keywords:"alpha rhythm"` in this case

**Comment by waxlamp** [comment]
> notes:
> 
> * it is sufficient to "enter" into searchbar in the original dataset landing page -- it would jump into shown above search results

I don't understand what this means--could you clarify?

**Comment by yarikoptic** [comment]
user clicks on `alpha rhythm` `<span>`, "onClick" event adds `"alpha rhythm"` into `///input` with id `input-115` ( ;-) -- would be nice to have an `id` like `top-search-bar`) and sends `.click()` or whatever is the event which is as if you hit "Enter".

**Comment by yarikoptic** [comment]
oh, may be I misunderstood the question.  What I meant is that it is sufficient to paste in a dandiset landing page search bar, e.g. straight on https://gui.dandiarchive.org/#/dandiset/000020 and when you hit "Enter" it jumps to https://gui.dandiarchive.org/#/dandiset/search?search=alpha

**Comment by waxlamp** [comment]
Why do we need to populate the search bar? Could we not simply link the, e.g., `alpha rhythm` chip directly to https://gui.dandiarchive.org/#/dandiset/search?search=alpha%20rhythm?

**Comment by yarikoptic** [comment]
Sure thing... not sure why I was talking in UI terms/elements, need vacation ;-)

**Comment by waxlamp** [comment]
Perhaps just some time in the sauna 😂

**Comment by sanuann** [comment]
@waxlamp is this done? if not, can I work on this?

**Comment by waxlamp** [comment]
> @waxlamp is this done?

I don't think so--you can always check by opening the deployed app and seeing whether the feature exists or not.

> if not, can I work on this?

Please do. I've assigned you to the issue.
